test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -10,6 +10,26 @@
 
 public class UtilsTest {
 
+    @Test
+    public void isAnyNull() {
```

Comment:
Recall from tutorial if the putting all test conditions within a Junit test is better or separating the test conditions into multiple Junit tests.